ulog/log.go

```diff
@@ -153,12 +155,17 @@ func (l *baseLogger) Log(lvl zap.Level, message string, keyVals ...interface{})
 	}
 }
 
+type stackTracer interface {
```

Comment:
Unfortunately, no: https://github.com/pkg/errors/blob/master/errors.go#L67
stack is a private struct.